packages/node_modules/@webex/plugin-meetings/src/meeting/index.js

```diff
@@ -1296,8 +1311,17 @@ export default class Meeting extends StatelessWebexPlugin {
    */
   setLocalTracks(localStream) {
     if (localStream) {
-      this.mediaProperties.setLocalAudioTrack(MeetingUtil.getTrack(localStream).audioTrack);
-      this.mediaProperties.setLocalVideoTrack(MeetingUtil.getTrack(localStream).videoTrack);
+      const {audioTrack} = MeetingUtil.getTrack(localStream);
```

Comment:
Instead of two lines, could you do:

```suggestion
      const {audioTrack, videoTrack} = MeetingUtil.getTrack(localStream);
```